readme.md

```diff
@@ -791,6 +791,7 @@
 - [Golem](https://github.com/golemfactory/awesome-golem#readme) - Open source peer-to-peer marketplace for computing resources.
 - [Stacks](https://github.com/friedger/awesome-stacks-chain#readme) - A smart contract platform secured by Bitcoin.
 - [Algorand](https://github.com/aorumbayev/awesome-algorand#readme) - An open-source, proof of stake blockchain and smart contract computing platform.
+- [Internet Computer](https://github.com/dfinity/awesome-icp#readme) - A public blockchain that hosts smart contracts that run at web speed.
```

Comment:
The repo title should be `awesome-internet-computer`. Same with the pull request title.

Reply:
I've changed the title to ICP. If I understand correctly, the issue is that the title here and the repo need to match.